bin/clone_local

```diff
@@ -19,15 +21,54 @@ OptionParser.new { |opts|
 }.parse!
 
 FileUtils.chdir Dir.pwd do
-  heroku_app_name = RadBin.get_heroku_app_name(:all)
+  prompt = TTY::Prompt.new
+
+  if options[:interactive]
```

Comment:
This all seems to be good, though there does seem to be a lot going on in this one file, not sure if there is a way we can either move some of it into RadBin or even just methods within this file. 

Maybe separate out the different functionality, like interactive, remote, local, etc. No really "right" way to do it, but files like this can start to get pretty big over time